lib/stage-autocompleter.js

```diff
@@ -121,9 +121,42 @@ class StageAutoCompleter {
    *
    * @returns {Function} The completion function.
    */
-  getCompletions(editor, session, position, prefix, done) {
+  async getCompletions(editor, session, position, prefix, done) {
     // Empty prefixes do not return results.
     if (prefix === '') return done(null, []);
+
+    // Comments block do not return results.
+    let value = session.getValue();
+    const findCommentsRe = new RegExp([
```

Comment:
Instead of creating the regex every time the hints are requested, I would move this to a constant so it's only created once at parse time.